src/projectscene/internal/projectsceneconfiguration.cpp

```diff
@@ -113,3 +117,25 @@ void ProjectSceneConfiguration::setIsEffectsPanelVisible(bool visible)
     }
     m_effectsPanelVisible.set(visible);
 }
+
+const std::unordered_map<std::string, std::string>& ProjectSceneConfiguration::clipColorMapping() const
+{
+    static std::unordered_map<std::string, std::string> colorMapping = {
+        { COLORS[0], "Blue" },
+        { COLORS[1], "Violet" },
+        { COLORS[2], "Magenta" },
+        { COLORS[3], "Red" },
+        { COLORS[4], "Orange" },
+        { COLORS[5], "Yellow" },
+        { COLORS[6], "Green" },
+        { COLORS[7], "Turquoise" },
+        { COLORS[8], "Cyan" }
+    };
```

Comment:
I think it's better to make one method that returns colors and keep the color names and their values ​​next to each other, something like this: 

```


const std::vector<std::pair<std::string/*name*/, std::string/*color*/>>& ProjectSceneConfiguration::clipColors() const
{
    static  std::vector<std::pair<std::string/*name*/, std::string/*color*/>> colors = {
        { "Blue", "#66A3FF"},
        { "Violet", "#9996FC"},
        ...
    };

    return colors;
}

We don't have to worry about searching, with such a small number of elements it doesn't matter whether it's a map or a linear search in a vector
```